src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[]{
+                new FigureSupplier().getRandomFigure(),
+                new FigureSupplier().getRandomFigure(),
+                new FigureSupplier().getRandomFigure(),
+                new FigureSupplier().getRandomFigure(),
+                new FigureSupplier().getDefaultFigure(),
+                new FigureSupplier().getDefaultFigure(),
+                new FigureSupplier().getDefaultFigure()
+        };
```

Comment:
let's create only one FigureSupplier object and then create empty array and use loop to fill it with random figures